system/LINK/etc/default/evm

```diff
@@ -21,6 +21,8 @@ export QPID_LOAD_MODULE=/usr/lib64/qpid/client/sslconnector.so
 # OpenStack server
 # - CFME does not currently support client ssl certificate for authentication
 export QPID_SSL_CERT_DB=/etc/qpid/certs
+# - location of the certificates
```

Comment:
This comment with the dash make it sound like a sub-bullet-point on the OpenStack server comment, which it isn't.  Can you update the comment to stand on its own?